storage/src/test/scala/coop/rchain/storage/StorageActionsTests.scala

```diff
@@ -203,6 +240,13 @@ abstract class StorageActionsTests extends FlatSpec with Matchers with OptionVal
     runK(r3)
 
     getK(r3).results should contain theSameElementsAs List(List("datum1", "datum2"))
+
+    store.withTxn(store.createTxnWrite()) { txn =>
+      store.removeJoin(txn, "ch1", List("ch1", "ch2"))
+      store.removeJoin(txn, "ch2", List("ch1", "ch2"))
```

Comment:
Are we doing this right?  Should there still be joins in the table at this point?